llvm/unittests/DebugInfo/DWARF/DWARFDieTest.cpp

```diff
@@ -839,4 +839,144 @@ TEST(DWARFDie, DWARFTypePrinterTest) {
   testAppendQualifiedName(Ctx->getDIEForOffset(0x1a), "t1<t3<int> >::t2");
   testAppendQualifiedName(Ctx->getDIEForOffset(0x28), "t3<int>::my_int");
 }
+
+TEST(DWARFDie, DWARFTypePrinterTest_BitInt) {
```

Comment:
Most of the testing for the printing roundtripping is happening in `llvm/test/tools/llvm-dwarfdump/X86/simplified-template-names.s` - could this testing be added there (& in the cross-project-tests that it depends on)

Reply:
Yea I was going to change those but realised they don't run on Darwin (not sure why). But I don't have a Linux machine handy to test it on. I could test it via PR CI (by trial and error).

Do you want me to remove the unit-test if we add it to `simplified-template-names.s`?